src/pokemon_icon.c

```diff
@@ -0,0 +1,1276 @@
+    1, // Shedinja
+    2, // Taillow
+    2, // Swellow
+    1, // Shroomish
+    1, // Breloom
+    1, // Spinda
+    0, // Wingull
+    0, // Pelipper
+    2, // Surskit
+    0, // Masquerain
+    2, // Wailmer
+    0, // Wailord
+    0, // Skitty
+    2, // Delcatty
+    1, // Kecleon
+    1, // Baltoy
+    0, // Claydol
+    0, // Nosepass
+    1, // Torkoal
+    2, // Sableye
+    0, // Barboach
+    0, // Whiscash
+    0, // Luvdisc
+    0, // Corphish
+    0, // Crawdaunt
+    2, // Feebas
+    0, // Milotic
+    0, // Carvanha
+    0, // Sharpedo
+    1, // Trapinch
+    1, // Vibrava
+    1, // Flygon
+    2, // Makuhita
+    1, // Hariyama
+    1, // Electrike
+    0, // Manectric
+    1, // Numel
+    0, // Camerupt
+    2, // Spheal
+    2, // Sealeo
+    0, // Walrein
+    1, // Cacnea
+    1, // Cacturne
+    2, // Snorunt
+    0, // Glalie
+    1, // Lunatone
+    0, // Solrock
+    2, // Azurill
+    0, // Spoink
+    2, // Grumpig
+    0, // Plusle
+    0, // Minun
+    2, // Mawile
+    0, // Meditite
+    0, // Medicham
+    0, // Swablu
+    0, // Altaria
+    0, // Wynaut
+    0, // Duskull
+    0, // Dusclops
+    0, // Roselia
+    2, // Slakoth
+    2, // Vigoroth
+    1, // Slaking
+    1, // Gulpin
+    2, // Swalot
+    1, // Tropius
+    0, // Whismur
+    2, // Loudred
+    2, // Exploud
+    0, // Clamperl
+    0, // Huntail
+    0, // Gorebyss
+    0, // Absol
+    0, // Shuppet
+    0, // Banette
+    2, // Seviper
+    0, // Zangoose
+    1, // Relicanth
+    2, // Aron
+    2, // Lairon
+    2, // Aggron
+    0, // Castform
+    0, // Volbeat
+    2, // Illumise
+    2, // Lileep
+    0, // Cradily
+    0, // Anorith
+    0, // Armaldo
+    1, // Ralts
+    1, // Kirlia
+    1, // Gardevoir
+    2, // Bagon
+    2, // Shelgon
+    0, // Salamence
+    0, // Beldum
+    0, // Metang
+    0, // Metagross
+    2, // Regirock
+    2, // Regice
+    2, // Registeel
+    2, // Kyogre
+    0, // Groudon
+    1, // Rayquaza
+    0, // Latias
+    2, // Latios
+    0, // Jirachi
+    0, // Deoxys
+    0, // Chimecho
+    1, // Egg
+    0, // Unown B
+    0, // Unown C
+    0, // Unown D
+    0, // Unown E
+    0, // Unown F
+    0, // Unown G
+    0, // Unown H
+    0, // Unown I
+    0, // Unown J
+    0, // Unown K
+    0, // Unown L
+    0, // Unown M
+    0, // Unown N
+    0, // Unown O
+    0, // Unown P
+    0, // Unown Q
+    0, // Unown R
+    0, // Unown S
+    0, // Unown T
+    0, // Unown U
+    0, // Unown V
+    0, // Unown W
+    0, // Unown X
+    0, // Unown Y
+    0, // Unown Z
+    0, // Unown Exclamation Mark
+    0, // Unown Question Mark
+};
+
+const struct SpritePalette gMonIconPaletteTable[] =
+{
+    { &gMonIconPalettes[0][0], POKE_ICON_BASE_PAL_TAG + 0 },
+    { &gMonIconPalettes[1][0], POKE_ICON_BASE_PAL_TAG + 1 },
+    { &gMonIconPalettes[2][0], POKE_ICON_BASE_PAL_TAG + 2 },
+
+// These palettes are actually part of the title screen BG palette and are not
+// used for Pokémon icons.
```

Comment:
That comment was copied from another file. I think it refers to Emerald rather than Ruby/Sapphire. https://github.com/pret/pokeemerald/blob/4db33778ad3faa64d994f46358a28c22c393f7c6/data/graphics.s#L1017